Testreglar/2.4.1/Nett/nett-2.4.1a.json

```diff
@@ -0,0 +1,501 @@
+					}
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10",
+					"delutfall": {
+						"nr": 2,
+						"fasit": "Nei",
+						"tekst": "<br>- ikke har tekst som er beskrivande"
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Aktiver snarveilenken.",
+			"ht": "",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				}
+			}
+		},
+		{
+			"stegnr": "3.8",
+			"spm": "Blir visuelt fokus flyttet til hovedinnholdet på nettsiden?",
+			"ht": "<p>Nettsiden skal rulle til hovedinnholdet, slik at starten på hovedinnholdet viser øverst i nettleseren.</p>\n<p><strong>Merk</strong>: Hovedinnholdet ligger vanligvis mellom topptekst (header) og bunntekst (footer).  </p>",
+			"type": "jaNei",
+			"kilde": [
+				"G1",
+				"G123"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.9",
+					"delutfall": {
+						"nr": 3,
+						"fasit": "Ja",
+						"tekst": "<br>- flytter visuelt fokus til hovedinnholdet ved aktivering"
+					}
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10",
+					"delutfall": {
+						"nr": 3,
+						"fasit": "Nei",
+						"tekst": "<br>- ikke flytter visuelt fokus til hovedinnholdet ved aktivering"
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.9",
+			"spm": "Blir tastaturfokus flyttet til det første aktive elementet i hovedinnholdet?",
+			"ht": "<p>Slik tester du: </p>\n<ul>\n<li>Trykk på tab-tasten igjen</li>\n<li>Sjekk om fokus flytter til første aktive elementet i hovedinnholdet\n<ul>\n<li>Hva som er første aktive element på en nettside, varierer. Det kan være en lenke, en video, et skjemaelement eller lignende. </li>\n</ul>\n</li>\n</ul>\n<p><strong>Tips:</strong> En vanlig feil, er at tastaturnavigeringen fortsetter fra toppen av nettsiden etter aktivering.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G1",
+				"G123"
+			],
+			"ruting": {
+				"ja": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"type": "talDersom",
+							"sjekk": [
+								"3.4",
+								"3.5",
+								"3.6",
+								"3.8",
+								"3.9"
+							],
+							"verdi": "Ja",
+							"mellom1": 5,
+							"mellom2": 5,
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Ja",
+								"utfall": "Det finnes en snarveilenke for å hoppe til hovedinnholdet, der snarveilenken<br>- ligger før blokken med innhold<br>- alltid er synlig eller blir synlig ved fokus<br>- har tekst som er beskrivande<br>- flytter visuelt og funksjonelt fokus til hovedinnholdet ved aktivering"
+							}
+						}
+					}
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10",
+					"delutfall": {
+						"nr": 4,
+						"fasit": "Nei",
+						"tekst": "<br>- ikke flytter funksjonelt fokus til hovedinnholdet ved aktivering"
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.10",
+			"spm": "Kan blokken med gjentatt innhold slås sammen, slik at du når hovedinnholdet, innen de fem første tab-stegene?",
```

Comment:
SCR28 gjelder bare meny. Er det hensiktsmessig å dele opp spørsmålet slik at utfallene blir enklere?

Reply:
Må uansett gjøre det klart at selve mekanismen for å collapse menyen er en del av de fem stegene.

Reply:
- Vi har oppdatert hjelpetekst i steg 3.1.1 med "innen de fem første tab-stegene". 
- Det er veldig krevende å lage kompleks ruting i en testregel der det skal sjekkes mange ting før vi lander et utfall med samsvar. Det er ikke hensiktsmessig å dele opp spørsmålet og gjøre rutingen i testregelen ende mer kompleks.